lang/python/python-dateutil/Makefile

```diff
@@ -15,33 +15,51 @@ PKG_LICENSE:=BSD-2-Clause
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/python-dateutil
 PKG_HASH:=88f9287c0174266bb0d8cedd395cfba9c58e87e5ad86b2ce58859bc11be3cf02
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-dateutil-$(PKG_VERSION)
 
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
this looks un-needed ; there is no host-side build here

Reply:
done